src/chat/flagsReducers.js

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
Could you explain why separating this into a separate reducer is a performance win? It seems like it will slow performance down to me